SourceX/display.cpp

```diff
@@ -296,15 +296,13 @@ SDL_Surface *CreateScaledSurface(SDL_Surface *src)
 } // namespace
 #endif // USE_SDL1
 
-void ScaleSurfaceToOutput(SDL_Surface **surface)
+SDLSurfaceUniquePtr ScaleSurfaceToOutput(SDLSurfaceUniquePtr surface)

```

Comment:
Old parameter was a double pointer, and now is a normal one. Is this ok?

Reply:
Yes